base/applications/cleanmgr/main.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * PROJECT:     ReactOS Disk Cleanup
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Main file
+ * COPYRIGHT:   Copyright 2020 Arnav Bhatt (arnavbhatt288 at gmail dot com)
+ */
+
+#include "precomp.h"
+
+UINT CleanmgrWindowMsg;
+
+int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance,
+    LPWSTR lpCmdLine, int nCmdShow)
+{
+    HANDLE hMutex = 0;
+    INITCOMMONCONTROLSEX InitControls;
+    INT_PTR DialogButtonSelect;
+    int nArgs = 0;
+    LPWSTR* ArgList = NULL;
+
+    UNREFERENCED_PARAMETER(hPrevInstance);
+    UNREFERENCED_PARAMETER(nCmdShow);
+    
+    /* Registering a window message for cleanmgr */
+    CleanmgrWindowMsg = RegisterWindowMessageW(L"CleanmgrCreated");
+    if (CleanmgrWindowMsg == 0)
+    {
+        DPRINT("RegisterWindowMessageW(): Failed to register a window message!\n");
+    }
+
+    InitControls.dwSize = sizeof(INITCOMMONCONTROLSEX);
+    InitControls.dwICC = ICC_TAB_CLASSES | ICC_LISTVIEW_CLASSES;
+    if (!InitCommonControlsEx(&InitControls))
+    {
+        DPRINT("InitCommonControlsEx(): Failed to register control classes!\n");
+        return FALSE;
+    }
+
+    /* Creating a mutex of this specific program to check if multiple instances are running */
+    CreateMutexW(NULL, FALSE, L"cleanmgr.exe");
+    if (hMutex)
```

Comment:
`hMutex` is null here, since you never assigned anything to it.

Reply:
@ThFabba I apologize for that, that was my big oversight.